mm2src/mm2_p2p/src/behaviours/atomicdex.rs

```diff
@@ -450,12 +459,12 @@ impl AtomicDexBehaviour {
                 self.spawn(future);
             },
             AdexBehaviourCmd::SendResponse { res, response_channel } => {
-                if let Err(response) = self
+                if let Err(PeerResponse::Err { err }) = self
```

Comment:
While running tests I noticed it sometimes prints errors like
`ERROR mm2_p2p::behaviours::atomicdex] Error sending response: Ok { res: [206, 103, 107, 3, 61] }`
I guess this PeerResponse enum contains not only errors

Reply:
It's an error saying you couldn't send `Ok { res: [206, 103, 107, 3, 61] }` through the channel.

Reply:
Yes, apparently it could be returned as Err(Ok()) if channel is closing (originally I thought channel errors would not happen in in-memory peer tests). 
I will revert this